ERROR - 2023-05-25 00:57:41 --> mysqli_sql_exception: Unknown column 'add_ons.id' in 'where clause' in C:\xampp\htdocs\ocake\ocake\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ocake\ocake\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `add_ons...', 0)
#1 C:\xampp\htdocs\ocake\ocake\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `add_ons...')
#2 C:\xampp\htdocs\ocake\ocake\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `add_ons...')
#3 C:\xampp\htdocs\ocake\ocake\system\Database\BaseBuilder.php(2079): CodeIgniter\Database\BaseConnection->query('UPDATE `add_ons...', Array, false)
#4 C:\xampp\htdocs\ocake\ocake\system\Model.php(338): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ocake\ocake\system\BaseModel.php(886): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\ocake\ocake\system\Model.php(693): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\ocake\ocake\app\Controllers\User.php(994): CodeIgniter\Model->update('13', Array)
#8 C:\xampp\htdocs\ocake\ocake\system\CodeIgniter.php(927): App\Controllers\User->placeorder()
#9 C:\xampp\htdocs\ocake\ocake\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\ocake\ocake\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ocake\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2023-05-25 00:58:36 --> mysqli_sql_exception: Unknown column 'add_ons.id' in 'where clause' in C:\xampp\htdocs\ocake\ocake\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ocake\ocake\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `add_ons...', 0)
#1 C:\xampp\htdocs\ocake\ocake\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `add_ons...')
#2 C:\xampp\htdocs\ocake\ocake\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `add_ons...')
#3 C:\xampp\htdocs\ocake\ocake\system\Database\BaseBuilder.php(2079): CodeIgniter\Database\BaseConnection->query('UPDATE `add_ons...', Array, false)
#4 C:\xampp\htdocs\ocake\ocake\system\Model.php(338): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ocake\ocake\system\BaseModel.php(886): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\ocake\ocake\system\Model.php(693): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\ocake\ocake\app\Controllers\User.php(994): CodeIgniter\Model->update('13', Array)
#8 C:\xampp\htdocs\ocake\ocake\system\CodeIgniter.php(927): App\Controllers\User->placeorder()
#9 C:\xampp\htdocs\ocake\ocake\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\ocake\ocake\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ocake\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2023-05-25 00:58:57 --> mysqli_sql_exception: Unknown column 'add_ons.id' in 'where clause' in C:\xampp\htdocs\ocake\ocake\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\ocake\ocake\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `add_ons...', 0)
#1 C:\xampp\htdocs\ocake\ocake\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `add_ons...')
#2 C:\xampp\htdocs\ocake\ocake\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `add_ons...')
#3 C:\xampp\htdocs\ocake\ocake\system\Database\BaseBuilder.php(2079): CodeIgniter\Database\BaseConnection->query('UPDATE `add_ons...', Array, false)
#4 C:\xampp\htdocs\ocake\ocake\system\Model.php(338): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\ocake\ocake\system\BaseModel.php(886): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\ocake\ocake\system\Model.php(693): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\ocake\ocake\app\Controllers\User.php(994): CodeIgniter\Model->update(13, Array)
#8 C:\xampp\htdocs\ocake\ocake\system\CodeIgniter.php(927): App\Controllers\User->placeorder()
#9 C:\xampp\htdocs\ocake\ocake\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\ocake\ocake\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ocake\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-05-25 09:27:23 --> Undefined index: user_id
in APPPATH\Controllers\User.php on line 975.
 1 APPPATH\Controllers\User.php(975): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index: user_id', 'APPPATH\\Controllers\\User.php', 975, [])
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\User->placeorder()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
